website/content/docs/agent/config/config-files.mdx

```diff
@@ -2198,3 +2198,16 @@ tls {
 Consul will not enable TLS for the HTTP or gRPC API unless the `https` port has
 been assigned a port number `> 0`. We recommend using `8501` for `https` as this
 default will automatically work with some tooling.
+
+## XDS Parameters
+
+- `xds` This is a nested object that configures how Consul serves the [xDS
+  protocol](https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol)
+  to Envoy and consul-dataplane.
+  - `rebalance_drain_rate` Configures the rate at which xDS sessions will be
+    drained to rebalance load between servers, in terminations per second.
+    Defaults to 10 per second.
+  - `rebalance_drain_burst` The size of the token bucket used to recharge the
+    xDS session drain rate limiter. Defaults to 50 tokens.
+    See https://en.wikipedia.org/wiki/Token_bucket for more details about how
+    token bucket rate limiters operate.
```

Comment:
I'm not particularly happy with these docs. I think we could do a better job of explaining how to choose the right bucket size vs rate.